examples/reactTemplates/LineInput.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Prism from '../js/prism';
+
+export default class LineInputDoc extends React.Component {
+    componentDidMount() {
+        Prism.highlightAll();
+    }
+
+    render() {
+        return (
+            <div id="line-input">
+                <h2 className="docs-page__h2">Line Input</h2>
+                <pre className="line-numbers">
+                    <code className="language-tsx">{`<LineInput> {InputContent} </LineInput>`}</code>
```

Comment:
The example is incorrect. The component doesn't accept children.

Reply:
It is accepting. `LineInput` is only wrapper. 
![image](https://user-images.githubusercontent.com/6507688/59616452-bdc27a00-9124-11e9-92a7-bfaed3e443a6.png)

Reply:
Sorry, my bad. I think I wanted to comment on `<Label>Some label</Label>` example, which doesn't accept children.